src/pages/fi-accounts/forms/AccountsForm.js

```diff
@@ -68,32 +61,47 @@ const AccountsForms = ({
             ...prevStore,
             recordId: res.recordId
           }))
-          formik.setFieldValue('recordId', res.recordId )
+          formik.setFieldValue('recordId', res.recordId)
```

Comment:
can we use async/await and remove then/catch blocks

Reply:
why? .then same result